embassy-nrf/src/nfct.rs

```diff
@@ -0,0 +1,409 @@
+//! NFC tag emulator driver.
+//!
+//! This driver implements support for emulating an ISO14443-3 card. Anticollision and selection
+//! are handled automatically in hardware, then the driver lets you receive and reply to
+//! raw ISO14443-3 frames in software.
+//!
+//! Higher layers such as ISO14443-4 aka ISO-DEP and ISO7816 must be handled on top
+//! in software.
+
+#![macro_use]
+
+use core::future::poll_fn;
+use core::sync::atomic::{compiler_fence, Ordering};
+use core::task::Poll;
+
+use embassy_hal_internal::{into_ref, PeripheralRef};
+use embassy_sync::waitqueue::AtomicWaker;
+pub use vals::{Bitframesdd as SddPat, Discardmode as DiscardMode};
+
+use crate::interrupt::InterruptExt;
+use crate::pac::nfct::vals;
```

Comment:
This part breaks on nrf5340 (it should be `nfct_s`).

Reply:
it's not anymore, make sure you're using the latest nrf-pac.